app/views/handlers/block.py

```diff
@@ -116,3 +117,11 @@ def _set_started_at_metadata(self):
             started_at = datetime.utcnow().isoformat()
             logger.info("Survey started", started_at=started_at)
             collection_metadata["started_at"] = started_at
+
+    def _get_safe_page_title(self, title):
+        if title and "placeholders" in title:
+            if "text_plural" in title:
+                title = title["text_plural"]["forms"]["other"]
```

Comment:
As the number is replaced with ... and other is always there, this seems the most logical one to use